README.md

```diff
@@ -20,6 +20,7 @@ This repository defines the following SAP vocabularies:
 - [PersonalData: Personal data / GDPR](vocabularies/PersonalData.md)
 - [Session: Sticky Sessions](vocabularies/Session.md)
 - [UI: Representing data in user interfaces](vocabularies/UI.md)
+- [Offline: Terms for Offline OData](vocabularies/Offline.md)
```

Comment:
```suggestion
- [Offline: Terms for Offline OData](vocabularies/Offline.md) - _experimental_
```

Reply:
We mark a vocabulary as "experimental" until at least one of its terms is used in globally available software products.